src/spring/azext_spring/_transformers.py

```diff
@@ -365,3 +365,12 @@ def transform_apm_type_output(result):
         item['Name'] = item['name']
 
     return result if is_list else result[0]
+
+
+def transform_support_server_versions_output(result):
+    is_list = isinstance(result, list)
+
+    if not is_list:
+        result = [result]
+
+    return result if is_list else result[0]
```

Comment:
From my understanding, this method does nothing. Do you wanna extract the important information from the raw data and show as table?

Reply:
defined table output columns